CMSgov/onemac
# OY2 - 11676: Package Dashboard: New Waiver tab layout

Story: https://qmacbis.atlassian.net/browse/OY2-11676 (AC #1 ONLY)
Endpoint: https://d196ml5yannhvf.cloudfront.net/

### Details
Data model and API call tweaking to support the changes to the Waivers Tab on the Package Dashboard.

### Changes
Created a bunch of ID helper functions to decode the waiver numbers and organize the components
Created the children attribute and sent that along with the other data for dashboard requests

### Implementation Notes
Even though the deeper structure is there for the components, it will probably need tweaking as we develop the front end

### Test Plan
1.  Log in as a submitter user
2. Go to packages page, Waivers tab - see both base waivers and renewals in the data
3. if you want, add a bunch of different waiver components, check the data (in dynamodb console or the network tab) to see the structure.
